src/components/Badge/Badge.scss

```diff
@@ -15,7 +15,7 @@ $pip-spacing: ($height - $pip-size) / 2;
   align-items: center;
   padding: $vertical-padding $horizontal-padding;
   background-color: var(--p-surface-neutral);
-  border: var(--p-override-zero);
+  border: 0;
```

Comment:
Can we remove the `border: 0` or is it overriding another value?

Reply:
I think we can remove it! I removed the property and then checked all Storybook examples in Chrome, Safari, and Firefox and didn't find anything that it might be overriding.

Reply:
I can go through all the other values and do the same. Let me know if you think I may be overlooking anything testing wise.

Reply:
Oh I just realized I didn't test on mobile (iOS or Android) if that's something else I should also do for all of these.

Reply:
Just make sure to test responsively if the CSS is inside a media query. We shouldn't need to device test these fallbacks.